sound/soc/sof/sof-audio.h

```diff
@@ -39,6 +39,8 @@
  */
 #define VOLUME_FWL	16
 
+#define SOF_TLV_ITEMS 3
```

Comment:
Can we reuse `TLV_ITEMS`? We may forget to change `TLV_ITEMS`  when we try to change `SOF_TLV_ITEMS` some day in the future.

Reply:
I see what you mean. Fixed now!